docs/developers-guide/cadl-docs/README.md

```diff
@@ -0,0 +1,119 @@
+# Morphir Type Mappings To CADL
+## SDK Types to CADL
+####### NB: CADL has a numeric type which consist of two subtypes called integer and float
+
+#### Basic Types
+###### Bool
+    This maps to the `boolean` type in CADL. 
+    Example: 
+        alias InCadl = boolean
+###### Int
+    In CADL, this maps to the subtype `integer`, which further has theses subtypes ::  
+    [-] int8        [-] uint8       [-] safeint
+    [-] int16       [-] uint16
+    [-] int32       [-] unit32
+    [-] int64       [-] unit32
+    
+    NB - To assign an `integer` type, you need to actually assign a subtype instead else the `integer` type defaults to an object. 
+
+    Example: 
+        alias length = int32 
```

Comment:
Please elaborate on the issue and specifically explain how integer is valid CADL but in OAS it will turn into object.

Reply:
done